### ISSUE TITLE
Attach Stewart platform to Jaguar

### ISSUE THREAD

**Issue by correll** [correll/advancedrobotics#5]

**Comment by bryanbocao** [comment]
Does anyone know where Stewart platform is in the lab? I could not find it in 1B15. Thanks!

**Comment by rmaheshkumarblr** [comment]
If I am not wrong, I think it is in Correll's Lab: 1B21

**Comment by correll** [comment]
It's in 1B21 on Andy McEvoy's desk.

**Comment by correll** [comment]
Before attaching it, we should probably finish up the code for it...

I'll be in on Tuesday and can go over where we are with the platform.

Andy McEvoy

Research Assistant
Correll Lab
Department of Computer Science
University of Colorado at Boulder

(832) 439 - 6150

**Comment by rcr2f** [comment]
Update from the recent meetings:

Steve suggested we should use 80/20 rail on the Jaguar rather than punching a bunch of holes in the top plate. The Husky robot is an example of how we should setup mounting. We also shouldn't have to order anything since there is extra in the lab.

http://www.8020.net/t-slot-5.asp

![img_20151002_141011941](https://cloud.githubusercontent.com/assets/6099168/10256638/d4f3eaa6-690f-11e5-86bf-56d8ef189077.jpg)
![img_20151002_141005817](https://cloud.githubusercontent.com/assets/6099168/10256639/d4f8a906-690f-11e5-8d3a-92261a1aba86.jpg)
![img_20151002_141026687](https://cloud.githubusercontent.com/assets/6099168/10256640/d4f96b84-690f-11e5-9edd-a08d470e7268.jpg)

**Comment by rcr2f** [comment]
More ideas on the 80/20....

![img_20151002_142306057](https://cloud.githubusercontent.com/assets/6099168/10256963/cd0f3410-6911-11e5-9a4c-1797334bed73.jpg)

**Comment by ryanleonard** [comment]
I just cut the railing, drilled holes for the screws, and hollowed out enough metal on the top layer to allow for the head of the screw to lay flush with the center of the railing. 

Professor Correll, Can you order some #7, 1.5in drywall screws so that we can attach the railing to the top plate?

**Comment by correll** [comment]
Hi Ryan

can you find this online and send me a link? We can also pick this up at
McGuckin.

Nikolaus

On Thu, Oct 15, 2015 at 3:04 PM, ryanleonard notifications@github.com
wrote:

> I just cut the railing, drilled holes for the screws, and hollowed out
> enough metal on the top layer to allow for the head of the screw to lay
> flush with the center of the railing.
> 
> Professor Correll, Can you order some #7
> https://github.com/correll/advancedrobotics/issues/7, 1.5in drywall
> screws so that we can attach the railing to the top plate?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/correll/advancedrobotics/issues/5#issuecomment-148520988
> .

## 

Assistant Professor
@correlllab, http://correll.cs.colorado.edu
Open robotics textbook: http://amzn.to/1vTjnt9 (buy), http://bit.ly/1vr7ptA
(contribute)

**Comment by correll** [comment]
We also have machine screws and washers here

On Thu, Oct 15, 2015 at 3:04 PM, ryanleonard notifications@github.com
wrote:

> I just cut the railing, drilled holes for the screws, and hollowed out
> enough metal on the top layer to allow for the head of the screw to lay
> flush with the center of the railing.
> 
> Professor Correll, Can you order some #7
> https://github.com/correll/advancedrobotics/issues/7, 1.5in drywall
> screws so that we can attach the railing to the top plate?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/correll/advancedrobotics/issues/5#issuecomment-148520988
> .

## 

Assistant Professor
@correlllab, http://correll.cs.colorado.edu
Open robotics textbook: http://amzn.to/1vTjnt9 (buy), http://bit.ly/1vr7ptA
(contribute)

**Comment by ryanleonard** [comment]
I picked up some screws last night, and just finished cutting the crossbars. Now we need to get some attachments compatible with 20mm x 20mm 8020. Everything that I could find in the lab was too big and wouldnt fit the railing.

**Comment by correll** [comment]
Let me know the mcmaster parts and i will order

On Fri, Oct 16, 2015 at 11:54 AM, ryanleonard notifications@github.com
wrote:

> I picked up some screws last night, and just finished cutting the
> crossbars. Now we need to get some attachments compatible with 20mm x 20mm
> 8020. Everything that I could find in the lab was too big and wouldnt fit
> the railing.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/correll/advancedrobotics/issues/5#issuecomment-148785252
> .

## 

Assistant Professor
@correlllab, http://correll.cs.colorado.edu
Open robotics textbook: http://amzn.to/1vTjnt9 (buy), http://bit.ly/1vr7ptA
(contribute)

**Comment by Zolkin1** [comment]
![mounts on jaguar](https://cloud.githubusercontent.com/assets/13017206/10591182/1cfcb408-7672-11e5-88d2-4378823cf5de.png)

Above is my current design for mounting the stewart platform based on what we discussed on Thursday. I raised the stewart platform 4 in above the jaguar top, that way the laser sensor can fit underneath. The laser sensor has three holes that should be able to mount into the 2 pieces of 80/20 in between the bottom of the stewart platform. I was not able to find the dimensions of the holes or any part of the laser sensor so the pieces of 80/20 might need to be shifted in a little. I am still trying to figure out how to mount the occam in the stewart platform while keeping it stable.
